memgpt/config.py

```diff
@@ -28,6 +28,10 @@
 
 model_choices = [
     questionary.Choice("gpt-4"),
+    questionary.Choice(
+        "gpt-4-turbo (developer preview)",
+        value="gpt-4-1106-preview",
+    ),
     questionary.Choice(
         "gpt-3.5-turbo (experimental! function-calling performance is not quite at the level of gpt-4 yet)",
```

Comment:
Should we make the choice not be `gpt-3.5-turbo`, but instead `gpt-3.5-turbo-16k`? I think it's generally available at this point, and non-16k is actually only 4k context limit which is really small for MemGPT.

Reply:
That sounds good to me.